vllm/core/block/prefix_caching_block.py

```diff
@@ -44,6 +47,9 @@ def __init__(
         # of self._cached_blocks.
         self._unused_cached_blocks: Dict[PrefixHash, BlockId] = {}
 
+        # A mapping of blockId to Block to track those cached blocks
+        self._blocks: Dict[BlockId, Block] = {}
```

Comment:
Is the intention to have this track all blocks allocated by the _hashless_allocator or just the blocks in _cached_blocks?

Reply:
It server for track mapping in the _cached_blocks, as mark_blocks_as_computed/get_common_computed_block_ids's func param past from block manager is block_id only, and it has no idea for underlying content hash